demo/locales/en.json

```diff
@@ -112,6 +112,7 @@
   "DEMO_IGNORE_MIN_BUFFER_TIME": "Ignore Min Buffer Time",
   "DEMO_INITIAL_SEGMENT_LIMIT": "Initial Segment Limit",
   "DEMO_DISABLE_AUDIO": "Disable Audio",
+  "DEMO_DISABLE_VIDEO": "Disable Video",
```

Comment:
You'll also have to add an entry for this configuration value to demo/locales/source.json, which is the file we will be showing to the translators once the demo localization system is finished.
I should probably add some sort of check in the build script that errors if you forget to do this, honestly. I definitely foresee other people forgetting in the future...

Reply:
Ok, I'll fix it tomorrow. Thanks!